README.md

```diff
@@ -36,15 +38,33 @@ docker run -p 80:8080 \
 cnwan_adaptor
 ```
 
+## Quickstart
+If you want a minimal working setup, the script ```examples/setup_kubecon_demo.sh``` sets everything up a for you in the adaptor and your SD-WAN controller. Before running it, please:
```

Comment:
You only need 3 backticks when you have multi-line code that you want to format with fixed width, and you want to optionally specify the syntax, linke bash, Python... For in-line short text 1 backtick is enough. Same for the other occurrences below.

Reply:
Done, thanks!